flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorTest.java

```diff
@@ -95,9 +95,9 @@ public void testDefaultPartitionName() throws Exception {
 		FileSystem fs = defaultPartPath.getFileSystem(hiveConf);
 		assertTrue(fs.exists(defaultPartPath));
 
-		TableImpl flinkTable = (TableImpl) tableEnv.sqlQuery("select * from db1.part order by x");
+		TableImpl flinkTable = (TableImpl) tableEnv.sqlQuery("select y, x from db1.part order by x");
```

Comment:
I noticed this test have a `testDifferentFormats` test, how about we parameterize this test class to these formats, and test all functionalities based on different formats?

Reply:
As far as the status quo is concerned, it is very difficult.
Because these tests are so slow. If we parameterize this test class, it will be very very slow...(Maybe will cause timeout)
I think maybe we should have a lightweight HiveCatalog in future. I created a JIRA to track it:
https://issues.apache.org/jira/browse/FLINK-15003

Reply:
BTW, this change just want to cover more for random select order.